### DISCUSSION TITLE
What is behind "Removed Spring framework support" note in 3.0.0 release?

### DISCUSSION THREAD

**Discussion by zjedi** [beanio/beanio#160] [Q&A] [answered]
Can you elaborate what is behind this note? Why removed, is it potentially not compatible with older versions?

**Comment by bjansen** [comment, marked as answer]
I have no idea, the original author made this decision 10 years ago and since then nobody was able to reach him :/

**Reply by bjansen** [reply]
See also https://github.com/beanio/beanio/discussions/135

**Reply by zjedi** [reply]
Aha, I thought this was new initiative in 3.0, thanks for clearing it up.
